src/main/java/com/teragrep/pth10/steps/teragrep/bloomfilter/BloomFilterTable.java

```diff
@@ -0,0 +1,136 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ *
+ * Additional permission under GNU Affero General Public License version 3
+ * section 7
+ *
+ * If you modify this Program, or any covered work, by linking or combining it
+ * with other code, such other code is not for that reason alone subject to any
+ * of the requirements of the GNU Affero GPL version 3 as long as this Program
+ * is the same Program as licensed from Suomen Kanuuna Oy without any additional
+ * modifications.
+ *
+ * Supplemented terms under GNU Affero General Public License version 3
+ * section 7
+ *
+ * Origin of the software must be attributed to Suomen Kanuuna Oy. Any modified
+ * versions must be marked as "Modified version of" The Program.
+ *
+ * Names of the licensors and authors may not be used for publicity purposes.
+ *
+ * No rights are granted for use of trade names, trademarks, or service marks
+ * which are in The Program if any.
+ *
+ * Licensee must indemnify licensors and authors for any liability that these
+ * contractual assumptions impose on licensors and authors.
+ *
+ * To the extent this program is licensed as part of the Commercial versions of
+ * Teragrep, the applicable Commercial License may apply to this file if you as
+ * a licensee so wish it.
+ */
+package com.teragrep.pth10.steps.teragrep.bloomfilter;
+
+import com.typesafe.config.Config;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.regex.Pattern;
+
+public final class BloomFilterTable {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(BloomFilterTable.class);
+    private final boolean ignoreConstraints;
+    private final FilterTypes filterTypes;
+    private final LazyConnection conn;
+
+    public BloomFilterTable(Config config) {
+        this(new FilterTypes(config), new LazyConnection(config), false);
+    }
+
+    // used in testing
+    public BloomFilterTable(Config config, boolean ignoreConstraints) {
+        this(new FilterTypes(config), new LazyConnection(config), ignoreConstraints);
+    }
+
+    public BloomFilterTable(FilterTypes filterTypes, LazyConnection conn, boolean ignoreConstraints) {
+        this.filterTypes = filterTypes;
+        this.conn = conn;
+        this.ignoreConstraints = ignoreConstraints;
+    }
+
+    public void create() {
+        if (ignoreConstraints && LOGGER.isDebugEnabled()) {
+            LOGGER.debug("Ignore database constraints active this should be only used in testing");
+        }
+        final String name = filterTypes.tableName();
+        final String sql = createTableSQL(name);
```

Comment:
something like new TableSQL(name).sql() would be preferable
(Create a new object) or change name to tableSql()

Reply:
Added CreateTableSQL class to handle sql and ignore constraints sql